src/client/java_gui/PiGui.java

```diff
@@ -290,7 +292,22 @@ public void doCompileAndMaybeSubmit(boolean alsoSubmit, String submissionComment
 	
 	public void doSubmit() {
 		new PiSubmit(this);
-	}	
+	}
+
+	public 	void doIncreaseFont() {
+		piCode.increaseFont();
```

Comment:
This line (and the similar one below) doesn't compile for me.  Am I missing something?  To get it to work, I had to add these methods to PiCode (and have them call the real methods that were implemented in its getPainter method).